packages/core/src/components/button/abstractButton.tsx

```diff
@@ -176,6 +177,11 @@ export abstract class AbstractButton<H extends React.HTMLAttributes<HTMLElement>
         this.props.onKeyUp?.(e);
     };
 
+    protected handleBlur = (e: React.FocusEvent<any>) => {
+        if (this.state.isActive) this.setState({ isActive: false });
```

Comment:
```suggestion
        if (this.state.isActive) {
            this.setState({ isActive: false });
        }
```

we used to have a lint rule for this 😕